compiler/rustc_attr/src/builtin.rs

```diff
@@ -463,7 +467,30 @@ pub fn cfg_matches(cfg: &ast::MetaItem, sess: &ParseSess, features: Option<&Feat
             }
             MetaItemKind::NameValue(..) | MetaItemKind::Word => {
                 let ident = cfg.ident().expect("multi-segment cfg predicate");
```

Comment:
```suggestion
                let name = cfg.ident().expect("multi-segment cfg predicate").name;
```
(Only the name is used below.)